gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<p>Does the Vertex AI API also need to be enabled?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Not that I'm aware of; don't see it in this notebook: https://github.com/googleapis/python-bigquery-dataframes/blob/main/notebooks/getting_started/ml_fundamentals_bq_dataframes.ipynb